sheetstorm/settings/base.py

```diff
@@ -33,14 +33,14 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'users',
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'django_countries',
-    'users',
```

Comment:
It's like I said in commit message
> The 'users' app should be befor 'django.contrib.admin' in base.py INSTALLED_APPS
in order to
> replace admin-style tempates with custom ones.
Otherwise all templates will have Django Admin view style (blue background and so on).

Reply:
The common way of organizing `installed_apps` is:
1. Django stuff.
2. 3rd party libraries stuff.
3. Application specific stuff.

Off course in alphabetic order ;) I need to understand the templates problem a little bit better, can you describe it? There should be a way of achieving that without a need to move this line.

Reply:
@kbeker @rwrzesien 
When I leave the common order of `INSTALLED_APPS`, then `django.contrib.admin` will be befor `users` app. Then for rendering password reset form and views will be used the default template (stored in `[python_version]/site-packages/allauth/templates/account`) instead of custom ones (created and stored in `users` app). 

I think that using `django.contrib.auth` Django is looking for the templates in default directory and when there are no ones, then in the main app directory... exactly like in  `INSTALLED_APPS` order.

Reply:
I will try to handle this, this should be doable.